CppSciComp/CppSciComp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,279 @@
+// !$*UTF8*$!
+{
+	archiveVersion = 1;
```

Comment:
gitignore this file and remove it from git.

To add files to git appropriately, on the command line
`git status` shows you what files have been modified
`git add <filename>` adds <filename> 
`git add -u <path>` adds all tracked files that have been modified but ignores ones not modified
`git add <path>` can inadvertently add junk files